src/main/kotlin/app/Config.kt

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app
+
+import app.utils.Options
+
+/**
+ * Config data class.
+ */
+data class Config (
```

Comment:
Yes, also, we discussed the ability of a user to modify the config file, I think it can't be done with protobuf.